app/views/pages/businessReliefs/BusinessReliefsNonDeductiblePageView.scala.html

```diff
@@ -0,0 +1,71 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.Layout
+@import views.html.templates.helpers.Heading
+@import views.html.templates.helpers.InputText
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.ErrorSummary
+@import views.html.templates.helpers.Link
+
+@import models.requests.AuthorisationRequest
+
+@this(
+      layout: Layout,
+      heading: Heading,
+      errorSummary: ErrorSummary,
+      govukWarningText: GovukWarningText,
+      formWithCsrf: FormWithCSRF,
+      inputText: InputText,
+      button: Button,
+      link: Link
+ )
+
+@(taxYear: Int, form: Form[BigDecimal])(implicit request: AuthorisationRequest[_], messages: Messages, appConfig: AppConfig)
+
+@layout(pageTitle = messages("business-reliefs.non-deductible.title"), hasErrors = form.hasErrors, taxYear = Some(taxYear), isAgent = request.user.isAgent) {
+
+    @errorSummary(form.errors)
+
+    @heading(messages("business-reliefs.non-deductible.title"), None, "govuk-!-margin-bottom-5")
+
+    @formWithCsrf(action = controllers.businessTaxReliefs.routes.BusinessReliefsNonDeductibleController.submit(taxYear)) {
+
+        <p class="govuk-body" id="non-deductible-paragraph1">@messages(s"business-reliefs.non-deductible.paragraph1.${if(request.user.isAgent) "agent" else "individual"}")</p>
+
+        <p class="govuk-body" id="non-deductible-paragraph2">@messages(s"business-reliefs.non-deductible.paragraph2.${if(request.user.isAgent) "agent" else "individual"}")</p>
+
+        <p class="govuk-body" id="non-deductible-paragraph3">
+            @messages(s"business-reliefs.non-deductible.paragraph3.1")
+            <a class="govuk-link" href=@{appConfig.selfAssessmentHS340Url}>@messages("business-reliefs.non-deductible.paragraph3.link")</a>
+            @messages(s"business-reliefs.non-deductible.paragraph3.3")
+            <strong>@messages(s"business-reliefs.non-deductible.paragraph3.4")</strong>
+        </p>
+
+        @inputText(
+            form = form,
+            id = "amount",
+            name = "amount",
+            currency = true,
+            label = messages("business-reliefs.non-deductible.question.input"),
+            labelClasses = Some("govuk-label--m"),
+            classes = Some("govuk-input--width-5")
```

Comment:
I don't think we should have this because the input width seems too small. Might be something we can check with the UX team.  

By default, for currency based on the help it would switch to `width-10` which feels better.

Reply:
Changed width as requested

![Input_Box](https://github.com/user-attachments/assets/65f94664-3083-430d-b6a6-dec7c2b9f6cf)